source/d3d11/depth_counter_tracker.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <d3d11.h>
+#include <unordered_map>
+
+class depth_counter_tracker
```

Comment:
Maybe rename this to `depth_stencil_tracker`? Also, it belongs in the `reshade::d3d11` namespace.

Reply:
Moved type to reshade::d3d11 namespace. 
`draw_call_tracker` is a better name indeed. Changed